docs/www/data-archiving.md

```diff
@@ -0,0 +1,95 @@
+    The random characters improve performance with S3 data transfers. 
+
+- Partition and topic manifests use prefixes with all symbols set to ‘0’ except the first one.
+    For example, a manifest path looks like `/00000000/meta/kafka/redpanda-test/topic_manifest.json` or `/00000000/meta/kafka/redpanda-test/6_2/manifest.json`.
+
+Redpanda adds the `rp-type` tag to all objects in S3:
+
+- Partition manifests - `rp-type=partition-manifest`
+- Log segments - `rp-type=segment`
+- Topic manifests - `rp-type=topic-manifest`
+
+These tags can be used to selectively delete data from S3.
+Both partition manifests and log segments can be safely deleted from S3 bucket when they get old (for instance, after one year).
+We recommend that you keep topic manifests in order to recover the corresponding topic.
+
+## Configuring data archiving
+
+1. Prepare the cloud storage:
+
+    - Amazon AWS S3
+
+        - Specify expiration rules for the files according to the tags.
+        - Use the IAM service to create a user to access S3.
+            - Grant this user permission to read and create objects.
+            - Copy the access key and secret key to the Redpanda configuration options `archival_storage_s3_access_key` and `archival_storage_s3_secret_key`, respectively.
+
+    - Google Cloud Storage
+
+        - Choose a uniform access control when you create the bucket.
+            - Use bucket level permissions for all objects.
+            - Use a Google managed encryption key.
+        - Create a service user with HMAC keys
+            and copy the keys to the Redpanda configuration options `archival_storage_s3_access_key` and `archival_storage_s3_secret_key`, respectively. 
+
+    > **_Note:_** The secret and access keys are stored in plain text in configuration files.
+
+2. Configure these parameters in the Redpanda configuration:
+
+        | Parameter name                                | Type         | Descripion                                              |
+        |-----------------------------------------------|--------------|---------------------------------------------------------|
+        | `archival_storage_enabled`                    | boolean      | Enables archival storage feature                        |
+        | `archival_storage_s3_access_key`              | string       | S3 access key                                           |
+        | `archival_storage_s3_secret_key`              | string       | S3 secret key                                           |
+        | `archival_storage_s3_region`                  | string       | AWS region                                              |
+        | `archival_storage_s3_bucket`                  | string       | S3 bucket                                               |
+        | `archival_storage_reconciliation_interval_ms` | milliseconds | Reconciliation period (default - 10s)                   |
```

Comment:
the type is integer - there is no such type as `milliseconds` the description should say 10000 (ms) is the default.